docs/docs/configuration/user-implemented-methods.mdx

```diff
@@ -22,6 +22,46 @@ public partial class CarMapper
 
 Whenever Mapperly needs a mapping from `TimeSpan` to `int` inside the `CarMapper` implementation, it will use the provided implementation.
 
+## Automatic user implemented mapping method discovery
+
+By default, user implemented mapping methods are discovered automatically.
+This can be disabled by setting `AutoUserMappings` to `false`.
+With disabled `AutoUserMappings`, only methods marked by the `UserMappingAttribute` hare discovered by Mapperly.
```

Comment:
hare -> are

Maybe also change
> With disabled `AutoUserMappings`

 to
 
 >With `AutoUserMappings` disabled
 
 Not quite sure which one is more correct

Reply:
Adjusted 🙈